web_scraping/steam_scraper/requirements.txt

```diff
@@ -1,3 +1,5 @@
+requests
```

Comment:
Why do you need requests?

Reply:
BeautifulSoup parses HTML from a string. This string can be obtained by making a GET request to the URL and returning the response text.

Reply:
This can also be handy for handling potential errors by checking for a 200 status code before proceeding with the data.